tests/test_crash_similarity.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import crash_similarity
+from crash_similarity_support import func, stack_trace, top_ten_stack_traces
+
+
+class CrashSimilarityTest(unittest.TestCase):
+
+    def test_clean_func(self):
+        for f in func:
+            resp = crash_similarity.clean_func(f)
+            self.assertTrue(resp.islower())
+            assert '\n' not in resp
+            if '@0x' in f:
+                self.assertEqual('@0x', resp[-3:])
```

Comment:
Can you check the entire output, instead of its properties?
```Python
funcs = [(func1, expected1), (func2, expected2), ...]
for f, expected in funcs:
    self.assertEqual(crash_similarity.clean_func(f), expected)
```